modules/@apostrophecms/asset/lib/webpack/src/webpack.scss.js

```diff
@@ -34,6 +35,7 @@ module.exports = (options, apos, srcBuildNames) => {
                 postcssOptions: {
                   plugins: [
                     [
+                      replaceViewportUnitsPlugin(),
```

Comment:
same here, the plugin should only be enabled if apos.asset.options.breakpointPreviewMode?.enable === true